### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by AliaksandraNovikava**
Please review =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,46 @@
-# New Next Project
+# AimHigh
 
-This project was created from the neue fische `next` template.
+![AimHigh](/public/aimhigh_app_frames.jpg)
```
Comment (MerleMak) [inline README.md:3]: This looks DOPE!! good job :))

File: README.md [hunk 2]
```diff
@@ -1,37 +1,46 @@
...
+- Next.js
+- React Testing Library
+- Jest
+- Styled Components
+- React DayPicker
+- LottieFiles
+- Material Design Icons
+- useLocalStorageState
+- confirmAlert
+- uid
```
Comment (MerleMak) [inline README.md:37]: Do you mean UUID?

Comment (MerleMak) [inline README.md:37]: I think i would seperate the tech stack into
- Highlights (React, Next.js, React Testing Library, Jest, Styled Components)
- Visuals (LottieFiles, Material Design Icons)
- Other (React) Libraries (React DayPicker, useLocalStorageState, confirmAlert, uuid)

If you wan't to be extra detailed you could also link the individual libraries like this:

- [LottieFiles](https://lottiefiles.com/)

But of course you can choose not keep it like it is, it's just a suggestion :)

**Review by MerleMak — COMMENTED** [review]
Love the Readme! Very nice Job 🚀⛵️

**Comment by AliaksandraNovikava** [comment]
Hi @MerleMak 
Thanks for your review☀️
I've adjusted my tech stack as you suggested.

**Pull request merged**